src/project_setup.py

```diff
@@ -61,30 +62,36 @@ def setup(
         func="model_server.py",
         name="llm-server",
         kind="serving",
-        image="gcr.io/iguazio/llm-serving:1.7.0",
+        image="gcr.io/iguazio/llm-serving:1.7.2",
         gpus=1,
     )
     _set_function(
         project=project,
         func="train.py",
         name="train",
         kind="job",
-        image="gcr.io/iguazio/monitoring-demo-adapters:1.7.0",
+        image="gcr.io/iguazio/monitoring-demo-adapters:1.7.2",
         gpus=1,
+        node_selector=node_selector,
+        node_name=node_name,
     )
     _set_function(
         project=project,
         func="metric_sample.py",
         name="metric-sample",
         kind="job",
         image="mlrun/mlrun",
+        node_selector=node_selector,
```

Comment:
Why add a node selector on non-gpu functions?

Reply:
see this [link](https://github.com/mlrun/demo-monitoring-and-feedback-loop/pull/4/files/2d211b23400c86f449ae1d8f226f7c4f283bd738#diff-20ca3499d2e5ebe2eb7dc5e06c3a664c33ea9948a7625e1b00c2dbe5859403f8R135) - `if gpus >= 1`.
it will add the node selector only for jobs that do not use GPU's